share/spice/jr_dev_jobs/jr_dev_jobs.js

```diff
@@ -0,0 +1,45 @@
+(function(env) {
+  'use strict';
+
+  env.ddg_spice_jr_dev_jobs = function(api_result) {
+    if (!api_result || !api_result.total_results > 0) {
+      return Spice.failed('jr_dev_jobs');
+    }
+
+    Spice.add({
+      id: 'jr_dev_jobs',
```

Comment:
nit: we ask that devs use 4-space soft tabs. Would you mind cleaning up the formatting here?